### DISCUSSION TITLE
Is there any extensions for alias directories?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#597] [Extension Show and Tell]
I use the [file-alias extension](https://marketplace.visualstudio.com/items?itemName=sumneko.file-alias) for aliasing filename,

![image](https://user-images.githubusercontent.com/12727582/236630157-ca62fe66-bf48-4a12-920a-63affc70ece1.png)

but it don't work for directory, 
is there any extensions for my requirement? and even whether it cannot be implemented by VSCode API?

**Comment by alefragnani** [comment]
Hi @liaodalin19903 ,

It seems the extension author expect it to work (https://github.com/sumneko/vscode-file-alias/issues/1). I suggest you to comment on that issue with your scenario, so the author could give you the right directions to make it work.

Hope this helps
